### PR TITLE
Upgrade MacID to 1.1 (14)

### PR DESCRIPTION
**Pull request by NikolayFrantsev**

### CODE REVIEW

File: Casks/macid.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask :v1 => 'macid' do
-  version '1'
-  sha256 '82b410b21599fe5a56011ab6152e91f5b8b17fb47924f2802abaaad2b373ae70'
+  version '1.1'
+  sha256 '80f9ac0b682f53b4c28c5958c0e4dddb3804aac31d48751109805865b9f3fbae'
 
-  url "http://macid.co/app/#{version}/6/MacID-for-OS-X.zip"
+  url "http://macid.co/app/#{version}/14/MacID-for-OS-X.zip"
```
Comment (vitorgalvao) [inline Casks/macid.rb]: Does this number (`14`, previously `6`) change with every release? Do you know what it signifies?

**Comment by Shutnik** [comment]
Seems it means build number, see http://macid.co/get/

File: Casks/macid.rb (hunk 1 above)
Comment (Shutnik) [inline Casks/macid.rb]: Seems it mean build number, see http://macid.co/get/

Comment (vitorgalvao) [inline Casks/macid.rb]: This needs an update, then, or having `#{version}` in `url` is useless.

Something like

```ruby
cask :v1 => 'macid' do
  version '1.1b14'
  sha256 '80f9ac0b682f53b4c28c5958c0e4dddb3804aac31d48751109805865b9f3fbae'

  url "http://macid.co/app/#{version.sub(%r{b.*},'')}/#{version.sub(%r{.*b},'')}/MacID-for-OS-X.zip"
  name 'MacID'
  homepage 'http://macid.co/'
  license :gratis

  app 'MacID.app'
end
```

Comment (Shutnik) [inline Casks/macid.rb]: Thanks, I had updated my commit with rebase.

**Pull request merged**